ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credentials.js

```diff
@@ -4,10 +4,6 @@
  */
 
 import Route from '@ember/routing/route';
-import { action } from '@ember/object';
-import { loading } from 'ember-loading';
-import { confirm } from 'core/decorators/confirm';
-import { notifySuccess, notifyError } from 'core/decorators/notify';
 import { inject as service } from '@ember/service';
```

Comment:
We are calling `@service router` within line 14, but I don't think is need? I don't see any usage of router within the route file.

Reply:
ahh good catch, thank you